core/storage/statistics/gae_models.py

```diff
@@ -400,10 +443,20 @@ def get_new_event_entity_id(cls, exp_id, session_id):
             exp_id,
             session_id))
 
+    # In the type annotation below, Dict[str, str] is used for 'params'.
+    # This is due to lack of information and this type of 'params' can be
+    # incorrect and can be changed in future.
```

Comment:
... This is due to lack of information about the possible values for `params`. If you're working with this part of the code in the future and find that the type for `params` is incorrect, please go ahead and change it, and feel free to remove this comment once you've done so.

Reply:
Done